scripts/converter/convert_to_enu.py

```diff
@@ -14,20 +14,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Convert world file from R2020a to R2020b keeping the NUE coordinate system."""
+"""Convert R2020b world file from the NUE to the ENU coordinate system."""
 
+import math
 import sys
 
 from transforms3d import quaternions
 
 from webots_parser import WebotsParser
-from converted_protos import converted_protos
 
 
-def rotation(value, r):
+def rotation(value, r, invert=False):
```

Comment:
I would remove this `invert` argument, as it is never used with the default value.